command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -158,11 +159,11 @@ def __init__(self, node_handle):
 
     @util.cancellableInlineCallbacks
     def _init(self):
-        self._moveto_action_client = yield action.ActionClient(self.nh, 'moveto', MoveToAction)
-        self._odom_sub = yield self.nh.subscribe('odom', Odometry)
-        self._trajectory_sub = yield self.nh.subscribe('trajectory', PoseTwistStamped)
-        self._trajectory_pub = yield self.nh.advertise('trajectory', PoseTwistStamped)
-        self._dvl_range_sub = yield self.nh.subscribe('dvl/range', Float64Stamped)
+        self._moveto_action_client = yield action.ActionClient(self.nh, '/moveto', MoveToAction)
```

Comment:
Whats the reason for this? Doesn't this just make the code less robust  to namespace changes?

For example, we run another small sub using the same stack and push all of these topic names down under the robot nam. I think this is the purpose of using relative names for topics.